### PR TITLE
Adds softshell jacket

### LINKED ISSUE #3343: $15 Bounty - Clothing request
Got a neat little sprite for a Nanotrasen branded softshell jacket, would like to see it in-game sometime relatively soon.
unknown
It's SFW, lore friendly, and doesn't clash with existing clothing sprites. What more could you want in a piece of clothing?
Let me list some of the characteristics I got in my head:

1. Open and Closed states (just visual changes)
2. Can hold certain tools and small items (essentially whatever the current Hazard Vest can hold)
3. Very weak armor (it's just cloth)

Payment can be done through PayPal only right now (only processor I have set up at the moment)
If you want to claim this bounty and make a PR for this item, hit me up on Discord at "thatguythere03#4418"

### PR DESCRIPTION
**Pull request by That0nePerson**
## About The Pull Request
Adds @thatguythere03's softshell jacket to wardrobe closets, the clothes vendor, and the beecoin shop. It can be toggled open and closed, and can hold the same items as the hazard vest. closes #3343 , and I'm doing it for free because it's very easy. Image:
image

## Why It's Good For The Game
Looks very very good, and adds some more varitey to wardrobes

## Changelog
:cl:That0nePerson, thatguythere03
add: Adds a softshell jacket
/:cl:

### CODE REVIEW

**Comment by utilbird** [comment]
> and I'm doing it for free because it's very easy

🥇

**Comment by Vexylius** [comment]
Why does it need voting? It's good, speedmerge it smh

**Comment by utilbird** [comment]
@Vexylius The laws I am binded to state everything that's not an internal code improvement need to be voted on

**Review by Crossedfall — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 1]
```diff
@@ -174,4 +174,8 @@
 	if(prob(30))
 		new /obj/item/clothing/suit/hooded/wintercoat(src)
 		new /obj/item/clothing/shoes/winterboots(src)
+	if(prob(40))
+		new /obj/item/clothing/suit/toggle/softshell(src)
+	if(prob(40))
+		new /obj/item/clothing/suit/toggle/softshell(src)
```
Comment (Crossedfall) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:180]: fuuuuck. Missed this somehow

**Pull request merged**